AzureCalling/app/src/main/res/layout/participant_view.xml

```diff
@@ -59,25 +59,39 @@
             android:visibility="invisible">
         </FrameLayout>
 
-        <ImageButton
+        <androidx.constraintlayout.widget.ConstraintLayout
             android:id="@+id/participant_switch_camera_button"
-            android:layout_width="24dp"
-            android:layout_height="24dp"
-            android:layout_marginTop="4dp"
-            android:layout_marginEnd="4dp"
-            android:layout_marginBottom="4dp"
-            android:background="@drawable/button_filled"
-            android:backgroundTint="#66000000"
-            android:contentDescription="@string/switch_camera_button"
-            android:paddingStart="2dp"
-            android:paddingTop="2dp"
-            android:paddingEnd="2dp"
-            android:paddingBottom="2dp"
+            android:layout_width="40dp"
+            android:layout_height="40dp"
+            android:background="@android:color/transparent"
             android:visibility="gone"
-            app:layout_constraintBottom_toBottomOf="parent"
+            android:clickable="true"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintTop_toTopOf="parent"
-            app:srcCompat="@drawable/ic_fluent_camera_switch_24_regular" />
+            >
+
+            <ImageButton
+                android:id="@+id/participant_switch_camera_image"
+                android:layout_width="24dp"
+                android:layout_height="24dp"
+                android:layout_marginStart="4dp"
+                android:layout_marginBottom="4dp"
+                android:background="@drawable/button_filled"
+                android:backgroundTint="#99000000"
+                android:contentDescription="@string/switch_camera_button"
+                android:paddingStart="2dp"
+                android:paddingTop="2dp"
+                android:paddingEnd="2dp"
+                android:paddingBottom="2dp"
+                android:clickable="false"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:srcCompat="@drawable/ic_fluent_camera_switch_24_regular">
+
+            </ImageButton>
```

Comment:
Since the parent is a clickable view, this does not need to be an ImageButton. We can use ImageView instead.